### PR TITLE
LPS-175001 Created maximum date and minimum date filter apply action

### PR DESCRIPTION
**Pull request by rafaelfs98**
[**LPS-173525**](https://issues.liferay.com/browse/LPS-173525)

### CODE REVIEW

**Comment by rafaelfs98** [comment]
ci:test:sf

**Review by Elvinho — APPROVED** [review]
🔥 LGTM 🔥

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/schema/filter.ts [hunk 1]
```diff
@@ -282,7 +289,9 @@ const filterSchema = {
 				type: 'date',
 			},
```
Comment (samuelleolopes) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/schema/filter.ts:293]: remove line

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Review by welitonpe — APPROVED** [review]
Great Job :fire:

File: modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/schema/filter.ts [hunk 2]
```diff
@@ -116,6 +120,20 @@ const baseFilters: Filter = {
 		},
 		type: 'select',
 	},
+	routine: {
+		label: i18n.translate('routines'),
+		name: 'routines',
+		options: [{label: 'Solutions', value: 'solutions'}],
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-testray/site-initializer-testray/extra/remote-app/src/schema/filter.ts:126]: No need to pass options if it comes from the API

```suggestion
```

**Review by kevenleone — CHANGES_REQUESTED** [review]
LGTM Rafael, but see my suggestions

**Comment by rafaelfs98** [comment]
@kevenleone  Fixed your suggestion

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Comment by kevenleone** [comment]
resent: https://github.com/pt-liferay-solutions/liferay-portal/pull/3262

**Pull request closed without merging**